docs/source/index.rst

```diff
@@ -6,17 +6,24 @@
 Substra documentation
 =====================
 
+Substra is an open source federated learning (FL) software. It provides a flexible Python library and a web application to run federated learning training at scale.
```

Comment:
"Software" is usually uncountable so this sentence is a bit weird. I usually say it is a federated learning framework, is that bad?

Reply:
we decided to use the word software and not framework a year ago. Let's discuss this more in person.